privaterelay/templates/includes/banners.html

```diff
@@ -1,7 +1,13 @@
+{% load static %}
 {% load relay_tags %}
 {% load ftl %}
 {% ftlconf bundle='privaterelay.ftl_bundles.main' %}
 
+{% if settings.DOMAIN_REGISTRATION_MODAL %}
+  {% block javascript %}
+    <script src="{% static 'js/domain-registration.js' %}"></script>
+  {% endblock %}
+{% endif %}
```

Comment:
Putting this feature behind a flag. If the ENV isn't set, the form just submits as expected (without any confirmation)